Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,6 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics * cms.SequencePlaceholder("mix"))
```

Comment:
Hi - finally getting through this PR

I would have thought that this change needs to propagate through elsewhere in CMSSW. Is it really unused?